examples/pyomobook/mpec-ch/ex1e.py

```diff
@@ -0,0 +1,16 @@
+# ex1e.py
```

Comment:
I stand corrected - the name of the file is also at the top of the `mpec-ch/ex` files.